tests/bsim/README.md

```diff
@@ -38,6 +38,13 @@ tests/bsim/net/sockets/echo_test/tests_scripts/echo_test_802154.sh
 ```
 
 
+Generating coverage reports
+---------------------------
+
+You can use tests/bsim/_generate_coverage_report.sh to generate a coverage report
+of your BabbleSim tests. Check its help for more info.
```

Comment:
```suggestion
of BabbleSim tests. Check its help for more info.
```
what if I want to know a coverage not only my tests :)

Reply:
rephrased :)